app/components/user_avatars_stack/index.tsx

```diff
@@ -0,0 +1,170 @@
+            alignItems: 'center',
+            width: size,
+            height: size,
+            borderWidth: (size / 2) + 1,
+            borderColor: theme.centerChannelBg,
+            backgroundColor: theme.centerChannelBg,
+            borderRadius: size / 2,
+            marginLeft: imgOverlap,
+        },
+        overflowContainer: {
+            justifyContent: 'center',
+            alignItems: 'center',
+            width: overflowSize,
+            height: overflowSize,
+            borderRadius: overflowSize / 2,
+            borderWidth: 1,
+            borderColor: theme.centerChannelBg,
+            backgroundColor: theme.centerChannelBg,
+            marginLeft: imgOverlap,
+        },
+        overflowItem: {
+            justifyContent: 'center',
+            alignItems: 'center',
+            width: overflowSize,
+            height: overflowSize,
+            borderRadius: overflowSize / 2,
+            borderWidth: 1,
+            borderColor: theme.centerChannelBg,
+            backgroundColor: changeOpacity(theme.centerChannelColor, 0.08),
+        },
+        overflowText: {
+            fontSize: 10,
+            fontWeight: 'bold',
+            color: changeOpacity(theme.centerChannelColor, 0.64),
+            textAlign: 'center',
+        },
+        listHeader: {
+            marginBottom: 12,
+        },
+        listHeaderText: {
+            color: changeOpacity(theme.centerChannelColor, 0.56),
+            fontSize: 12,
+            fontWeight: '600',
+        },
+    };
+});
+
+const UserAvatarsStack = ({breakAt = 3, currentUserId, style: baseContainerStyle, teammateNameDisplay, users}: Props) => {
+    const theme = useTheme();
+    const intl = useIntl();
+    const isTablet = useIsTablet();
+
+    const showParticipantsList = useCallback(preventDoubleTap(() => {
+        const renderContent = () => (
+            <>
+                {!isTablet && (
+                    <View style={style.listHeader}>
+                        <FormattedText
+                            id='mobile.participants.header'
+                            defaultMessage={'THREAD PARTICIPANTS'}
+                            style={style.listHeaderText}
+                        />
+                    </View>
+                )}
+                <UsersList
+                    currentUserId={currentUserId}
+                    teammateNameDisplay={teammateNameDisplay}
+                    theme={theme}
+                    users={users}
+                />
+            </>
+        );
+
+        bottomSheet({
+            closeButtonId: 'close-set-user-status',
+            renderContent,
+            snapPoints: [(Math.min(14, users.length) + 3) * 40, 10],
+            title: intl.formatMessage({id: 'mobile.participants.header', defaultMessage: 'THREAD PARTICIPANTS'}),
+            theme,
+        });
+    }), [isTablet, teammateNameDisplay, theme, users]);
+
+    const displayUsers = users.slice(0, breakAt);
```

Comment:
this will in fact show 4 users instead of 3 as the value of `breakAt` suggest. My recommendation here is to:
1. change the prop name to `numOfUsersToDisplay` and then set it to the actual number you want to display, then here in the code you do `const displayUsers = users.slice(0, numOfUsersToDisplay - 1)`.

Reply:
It shows first 3 users - `users.slice(0, 3)`